### ISSUE TITLE
Flickering text with linum-relative

### ISSUE THREAD

**Issue by Pireax** [coldnew/linum-relative#23]
Hello, 
I really like this package but I have this issue where if I enable linum-relative sometimes the text will flicker when moving vertically. I found someone else who also had this issue and he had a video of it on youtube: https://www.youtube.com/watch?v=7-V-qXoeZ1w.

**Comment by coldnew** [comment]
@Pireax Do you have this issue when running `linum-mode` only ?

**Comment by Pireax** [comment]
No, it works fine using linum mode only.

**Comment by coldnew** [comment]
Hmm... it's werid. `linum-relative` work on my MacOSX 10.10.5 with emacs 24.5.1 without flikering.
Which OS and emacs do you use ?

I found there's another flickering issue on reddit: https://www.reddit.com/r/emacs/comments/2r89ny/

**Comment by Pireax** [comment]
Yeah that actually the dude the video's from! I use Windows 8.1 with Emacs 24.5.

**Comment by aldanor** [comment]
Can confirm, there's *massive* flickering on Windows when using `linum-relative`, any idea why? This seems to be the only package I have that causes emacs to flicker so much when scrolling.

// On my OSX machine there's no problem.

**Comment by debuuu** [comment]
Also getting huge flickering, on Windows 10, using only the relative line numbering - normal line numbering with linum works fine. I also get this flickering when using the other popular relative line numbering package for Emacs.

**Comment by zv** [comment]
I'm on Linux, considerable flickering with relative line numbers.

**Comment by azjgard** [comment]
On Windows 7, also considerable flickering with relative line numbers.

**Comment by amosbird** [comment]
This also happens to builtin relative linum in terminal emacs.

```
(setq display-line-numbers 'relative)
```

**Comment by coldnew** [comment]
@amosbird Which operating system do you use when has flickering text issue with relative linum? Windows?

**Comment by amosbird** [comment]
It's running in a remote CensOS 7 box. The host uses ssh and alacritty.
